OverlayPlugin.Core/EventSources/MiniParseEventSource.cs

```diff
@@ -138,6 +142,75 @@ public MiniParseEventSource(ILogger logger) : base(logger)
                 });
             });
 
+            RegisterEventHandler("nextLineMatching", (msg) => {
+                if (!msg.ContainsKey("rseq"))
+                    return null;
+                var rseq = msg["rseq"].ToObject<int>();
```

Comment:
I don't see a point in using `rseq` here. It's confusing since the naming overlaps with the `rseq` name in other places although it doesn't have anything to do with that other property.
It's also never really used anywhere and might even cause severe problems since two overlays using the same `rseq` would cause conflicts.